billing/views.py

```
@@ -0,0 +1,29 @@
+                        'product_data': {'name': 'Premium Plan'},
+                        'unit_amount': 1000,
+                    },
+                    'quantity': 1,
+                }],
+                mode='subscription',
+                success_url='http://localhost:5173/success',
+                cancel_url='http://localhost:5173/cancel',
+                metadata={'user_id': request.user.id}
+            )
+            return Response({'url': session.url})
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)